language/move-compiler/src/cfgir/cfg.rs

```diff
@@ -25,6 +28,12 @@ pub trait CFG {
     fn commands<'a>(&'a self, label: Label) -> Box<dyn Iterator<Item = (usize, &'a Command)> + 'a>;
     fn num_blocks(&self) -> usize;
     fn start_block(&self) -> Label;
+
+    fn next_block(&self, label: Label) -> Option<Label>;
+
+    fn last_continue_to_loop(&self, label: Label) -> Option<Label>;
```

Comment:
Can you formally to define this `last_continue_to_loop` concept in terms of nodes and edges in the CFG (and perhaps reify this explanation in a comment?). I think I asked this elsewhere--apologies if I missed the answer. But I think it's pretty important for someone trying to read/modify this code to understand the concept in depth.

Reply:
Would this be an appropriate enough comment:
"Blocks have their labels generated in ascending order based on the position in the original source text. This means that blocks with higher numerically valued labels appear later in the CFG than those with lower valued labels. The "last continue" of the loop is the back edge with the highest numerical value. In other words, it represents the "end" of the loop as the loop appeared in the source text (and in turn in the CFG)